blog/docs/releases/announcing-knative-v1-3-release.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "v1.3 release"
+linkTitle: "v1.3 release"
+Author: "Samia Nneji"
+Author handle: https://github.com/snneji
+date: 2022-03-08
+description: "Knative v1.3 release announcement"
+type: "blog"
+---
+
+## Announcing Knative v1.3 Release
+
+A new version of Knative is now available across multiple components.
+
+Follow the instructions in the documentation
+[Installing Knative](https://knative.dev/docs/install/) for the respective component.
```

Comment:
```suggestion
[Installing Knative](https://knative.dev/docs/install/) for the respective components.
```

(the plural sounds better to my ear, but not 100% that it's grammatically correct...)

Reply:
I think this is referring to installing individual components, but the wording could be clearer. I'll update this.